subprojects/platform-jvm/src/main/java/org/gradle/jvm/toolchain/internal/DefaultJavaLanguageVersion.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.jvm.toolchain.internal;
+
+import org.gradle.jvm.toolchain.JavaLanguageVersion;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+public class DefaultJavaLanguageVersion implements JavaLanguageVersion, Serializable {
+
+    private final int version;
+
+    public DefaultJavaLanguageVersion(int version) {
+        this.version = version;
+    }
+    @Override
+    public int asInt() {
+        return 0;
```

Comment:
:x: This is obviously wrong, missing test coverage maybe?

Reply:
Woops, clearly, fixing!